pkg/iss/iss.go

```diff
@@ -541,6 +558,11 @@ func (iss *ISS) applyStableCheckpointMessage(chkpPb *checkpointpb.StableCheckpoi
 
 	iss.logger.Log(logging.LevelDebug, "Installing state snapshot.", "epoch", chkp.Epoch())
 
+	result, err := issutil.LeaderPolicyFromBytes(chkp.Snapshot.EpochData.LeaderPolicy)
+	if err != nil {
+		iss.logger.Log(logging.LevelWarn, "Error deserializing leader selection policy from checkpoint", err)
+	}
```

Comment:
We also need to return an empty event list here, so the malformed message is ignored.

Reply:
Done.